src/components/PopupTrigger.js

```diff
@@ -25,7 +25,7 @@ const styles = {
 export const Popup = onClickOutside(function({ id, side = 'right', target: targetId, onClick, children, popupProps = {} }) {
   // We're passing popupProps here rather than just props, because ...props also includes lots of internal onClickOutside properties which
   // aren't valid to be dropped on a DOM element.
-  Utils.useConsoleAssert(popupProps['aria-label'] || popupProps['aria-labelledby'], 'In order to be accessible, Popup needs a label')
+  Utils.useLabelAssert('Popup', { ...popupProps })
```

Comment:
Any reason we need this spread? Or can we just pass `popupProps`?

Reply:
Yep, I was doing this everywhere since some also have other properties. But you're right about this one.